src/constants.js

```diff
@@ -6,5 +6,8 @@ export default {
     copy: {
```

Comment:
Signal was part of the copy that I moved across to the constants. The variable is only used in a message when `Store.netMgr.online` is `false` -- check Tap/Scan. Likewise `VOUCHER_SUBMIT_INVALID` is used only when client side validation fails.